### ISSUE TITLE
Option to always query for database password

### ISSUE THREAD

**Issue by Deathsbreed** [pwman3/pwman3#47]
Basically, I would like to have pwman3 always open in a terminal window, but that would be insecure because then someone just has to come around to my laptop when I'm away for a bit and they have full access to my passwds. Solution? Always query for the master password, or at least for important commands like `cp` or `p`.

Also, if this features already exists, please add it to the docs (that's where I've been looking).

**Comment by oz123** [comment]
```
pwman3 0.9.1 (c) visit: http://pwman3.github.io/pwman3/
pwman> ls

ID  USER                     URL                      TAGS                
Please type in your master password:
1   oz                        server            private               
 ...              
72  oz.tiram                 laptop                work                
pwman> h

Documented commands (type help <topic>):
========================================
cls   delete  exit    forget  info  new   passwd  tags
copy  edit    export  help    list  open  print 

Aliases:
========
EOF  cp  e  h  ls  n  o  p  rm

pwman> help forget
Usage: forget
Forgets the database password. Your password will need to  be reentered before accessing the database again.
pwman> forget
pwman> ls

ID  USER                     URL                      TAGS                
Please type in your master password:
```

You are correct it's missing in the tutorial, also `forget` is not as clear as `lock`.

**Comment by Deathsbreed** [comment]
I too found this command after having posted, although even then it would be nice to have a setting so that it will forget the master key every X minutes/seconds or even every time you read from the database. So it still does not necessarily add to what I was saying (mostly because saying `forget` every time I use a command would probably be a hassle).

Either way, this'll do for now. Thanks.

**Comment by oz123** [comment]
Actually there is an option to expire the password after a while, but you are right it is not well documented.

**Comment by oz123** [comment]
OK, the option was also buggy. I needed to fix it. Expect a 0.9.2 release soon.

**Comment by Deathsbreed** [comment]
@oz123 thanks... also, where is this option documented? I looked at <https://pwman3.readthedocs.io/en/latest/>, but I couldn't find anything (at least not in the "Configuring Pwman3" section, where I suppose it would be). Just so when the new release comes about (mostly when it comes about via Debian repos, which is where I get it from) I can configure it.

**Comment by oz123** [comment]
@Deathsbreed unfortunately, the package in the debian repos  is very undated. 
The maintainer does not upload any new package. Maybe you should open a bug about
that.
Other than that, use `pip install -U pwman3` to get the latest package.
I will post you when the package was uploaded.

**Comment by oz123** [comment]
The issue is fixed in version 0.9.2. The version can be found here or in PyPI.
